include/c/sksg_invalidation_controller.h

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2014 Google Inc.
+ * Copyright 2015 Xamarin Inc.
+ * Copyright 2017 Microsoft Corporation. All rights reserved.
+ *
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+
+#ifndef sksg_invalidationcontroller_DEFINED
+#define sksg_invalidationcontroller_DEFINED
+
+#include "include/c/sk_types.h"
+
+SK_C_PLUS_PLUS_BEGIN_GUARD
+
+SK_C_API sksg_invalidation_controller_t* sksg_invalidation_controller_new();
```

Comment:
I see a new, but no delete. Is this ever deleted? This is a POCO class, so will need an explicit delete because the normal unref will probably throw or do something crazy.